build.gradle

```diff
@@ -57,47 +63,28 @@ subprojects {
   apply plugin: 'java'
   apply plugin: 'maven' // make pom files for deployment
   apply plugin: 'nebula.maven-base-publish'
-  repositories {
-    mavenCentral()
-    mavenLocal()
-    maven { url  "http://palantir.bintray.com/releases" }
-  }
 
   configurations {
     testCompile.extendsFrom compileOnly
     all {
-      resolutionStrategy {
-        force 'org.apache.httpcomponents:httpclient:4.4.1'
-      }
       exclude group: 'org.slf4j', module: 'slf4j-log4j12'
     }
   }
 
   ext {
-    hadoopVersion = '2.7.3'
-    avroVersion = '1.8.2'
-    orcVersion = '1.5.5'
-    parquetVersion = '1.10.0'
-    hiveVersion = '1.2.1'
-
-    jacksonVersion = '2.6.7'
-
-    scalaVersion = '2.11'
-    sparkVersion = '2.4.0'
-    caffeineVersion = "2.7.0"
     jmhVersion = '1.21'
   }
 
   sourceCompatibility = '1.8'
   targetCompatibility = '1.8'
 
   dependencies {
-    compileOnly 'org.slf4j:slf4j-api:1.7.5'
-    compileOnly 'com.google.guava:guava:28.0-jre'
+    compileOnly 'org.slf4j:slf4j-api'
+    compileOnly 'com.google.guava:guava'
 
-    testCompile 'junit:junit:4.12'
-    testCompile 'org.slf4j:slf4j-simple:1.7.5'
-    testCompile 'org.mockito:mockito-core:1.10.19'
+    testCompile 'junit:junit'
+    testCompile 'org.slf4j:slf4j-simple'
+    testCompile 'org.mockito:mockito-all'
```

Comment:
Why did this need to change?

Reply:
I think Mockito-all was changed to mockito-core by @aokolnychyi  since mockito-all bundled classes which conflicted with junit https://github.com/apache/incubator-iceberg/pull/228